mobile/appcontainer/VersionBar.ts

```diff
@@ -15,8 +15,8 @@ import './VersionBar.scss';
  */
 export const versionBar = hoistCmp.factory({
 
-    render() {
-        if (!isShowing()) return null;
+    render: function () {
```

Comment:
prefer the more compact original specification of `render() {`  vs. `render: function() {`.  I assume this was unintentional?